tests/cypress/integration/Site-Capabilities/siteCapabilities.cy.js

```diff
@@ -0,0 +1,172 @@
+import { GetPluginId, getAppId } from '../wp-module-support/pluginID.cy';
```

Comment:
@sangeetha-nayak Can we please make below change as suggested by @circlecube 
`we're not writing cypress tests that check for text on the screen in favor of tests that check that elements are visible. This would make tests more maintainable since we won't need to adjust per brand. This goes along with the idea of testing critical paths in cypress. `

Reply:
Done, please check.